src/rTerminal.ts

```diff
@@ -72,6 +72,11 @@ export async function chooseTerminal() {
         return window.activeTerminal;
     }
 
+    let msg = '[chooseTerminal] ';
+    msg += `A. There are ${window.terminals.length} terminals: `;
```

Comment:
What is this `A.` and 96 lines `B.`, and `C`?
If it means defects depth, how is using the indent?

Reply:
Sorry, I should have added more info to the PR. My theory is that the error may be occurring because the number of terminals changes during that section of code. The `A`, `B` and `C` are so we can check the numbers and names of the terminals are three different points. We can see if they are the same each time or if they are changing.

Reply:
Thank you for the detail. It is clear.
In the future,  we may need to define our extension error code such as `chooseTerminalA` when we use a similar process to identify the point of the defect,